source/developers-guide/plugin-system/index.md

```diff
@@ -706,13 +706,13 @@ Backend plugin configuration can be extended by `Resources/config.xml` file. Thi
 To read out the configuration of your plugin use this code snippet in your base plugin class:
 
 ```php
-$config = $this->container->get('shopware.plugin.config_reader')->getByPluginName($this->getName());
+$config = $this->container->get('shopware.plugin.cached_config_reader')->getByPluginName($this->getName());
 ```
 Use it like this in other places:
 ```php
-$config = Shopware()->Container()->get('shopware.plugin.config_reader')->getByPluginName('SwagSloganOfTheDay');
+$config = $this->container->get('shopware.plugin.cached_config_reader')->getByPluginName('SwagSloganOfTheDay');
```

Comment:
the container property is not available in every class by default, that's why we used the `Shopware()` singleton in this example